### ISSUE TITLE
Transparency issue in Cinnamon over Arch

### ISSUE THREAD

**Issue by ykonstant1** [abishekvashok/cmatrix#89]
I am submitting a  Bug Report

### Problem or Feature in Brief
I am using the latest arch bundled version of cmatrix (2.0) with Cinnamon DE in Arch.  My terminal emulators include Terminator, urxvt, Gnome term and Konsole.  They are configured with light yellow tranparent background.  When cmatrix runs, it replaces that background with black in all samples.  After I exit cmatrix, transparency is restored.

On my Mint system with Cinnamon DE, with the exact same stylistic choices and terminals, cmatrix (v. 1.2 bundled with Ubuntu) correctly respects the transparent terminal background.

It is unclear to me where the bug lies, but since I and another couple of people could not find any help in linux fora, I hope the developer may be able to help.

Edit: Downgrading the package to the 1.2 version fixes the problem.  There seems to be something wrong in the transition 1.2 to 2.0 for my system.

**Comment by abishekvashok** [comment]
Strange, will let you know if I find any information

**Comment by eromatiya** [comment]
Same problem.  Using arch linux, black background in kitty terminal with transparency. Fixed by installing `cmatrix-git` in aur that uses cmatrix 1.2.

**Comment by leonardopn** [comment]
same problem in manjaro gnome

**Comment by joan31** [comment]
Same bug on arch.
I'm on i3wm and termite.

**Comment by xiaoyu2006** [comment]
<img width="1172" alt="截屏2020-07-28 11 01 23" src="https://user-images.githubusercontent.com/32544798/88614135-ae508700-d0c1-11ea-8095-47a23ebd13c4.png">

On macOS.

**Comment by AnirvanB** [comment]
Also on Arch.
Seeing an opaque background. DE: GNOME and Terminal. Downgrading to cmatrix 1.2 (installing `cmatrix-git` from AUR).

![WhatsApp Image 2020-08-16 at 12 27 11 PM](https://user-images.githubusercontent.com/44327359/90333129-b2950380-dfe0-11ea-9242-9b4ebd4839ac.jpeg)

**Comment by joan31** [comment]
Same on Konsole and Plasma 5

**Comment by space-pagan** [comment]
This is most likely due to an issue in the cmake directive. If the anyone here would like to try building with 
```
autoreconf -i
./configure
make
```
and let me know the results, it would be appreciated.

**Comment by xlucn** [comment]
@space-pagan Confirmed on Arch Linux, building with `make` gives the default background color, and with `cmake` does not.

Also, the difference between aur package and official repo package is exactly the build method they use, see
https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=cmatrix-git#n23
https://github.com/archlinux/svntogit-community/blob/ad595eb1c0a7b67f8b8652cd4ad8ca9d0ea5cb1a/trunk/PKGBUILD#L26

**Comment by space-pagan** [comment]
Wonderful. I'm working on fixing the cmake to make it behave the same as autotools. Expect a patch soon.

**Comment by joan31** [comment]
Transparency issue is still present.
It work with AUR package, but not official repo package.

**Comment by joan31** [comment]
Bug is still here, so I've opened a bug track on archlinux :
https://bugs.archlinux.org/task/76925

Comments and upvote to move resolution forward. :)

**Comment by gqtn** [comment]
> This is most likely due to an issue in the cmake directive. If the anyone here would like to try building with
> 
> ```
> autoreconf -i
> ./configure
> make
> ```
> 
> and let me know the results, it would be appreciated.

Yes, I also can confirm it. Building with `make` gave me the transparency. Thank you.
